portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACPagePath.macro

```diff
@@ -0,0 +1,54 @@
+definition {
+
+	@summary = "Clears segment search filter"
+	macro clearSegmentSearchFilter() {
+		Type.sendKeys(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "keys=CONTROL,a");
+
+		KeyPress(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "\BACK_SPACE");
+	}
+
+	@summary = "Selects the segment in page filter"
+	macro clickFilterSegment(segmentName = null) {
+		Click(
+			key_segmentName = ${segmentName},
```

Comment:
Hey @joshitagaki, I think this should be the name of the variable to work with the xpath you created, right?

```suggestion
			key_nameList = ${segmentName},
```